rails-session.cabal

```diff
@@ -40,6 +48,7 @@ test-suite specs
   build-depends:       base
                      , bytestring
                      , filepath
+                     , semigroups
```

Comment:
If all you are using `semigroup` for is for `NonEmpty`, I think we can either add some `CPP` pragmas or decide whether or not we want to support older GHCs, as `base-4.6` (I think) onwards ships with `Data.List.NonEmpty` already. Not super fussed, though.

Reply:
Will try to make this conditional. Does the repo have CI setup to test across different versions?

Reply:
Not at present https://github.com/iconnect/rails-session/issues/3

Reply:
There's now CI and a build matrix that tests against GHC 7.10 and 8.0. 

If you pull in `master` it should work against your PR.

Reply:
Thanks @filib .